test/cors-error.js

```diff
@@ -0,0 +1,30 @@
+const { expect } = require('chai')
+const CorsError = require('../lib/cors-error')
+
+describe('cors error object', () => {
```

Comment:
Since this is a library, we only want to test the interface: inputs and expected outputs.  Implementation tests are inherently brittle since they block any refactors, and don't add any value once you've tested the interface.